### PR TITLE
Transifex

### PR DESCRIPTION
**Pull request by jospoortvliet**
This should set our site up for translation. It will also break the open pullrequests - the team page and the enterprise page changes. I'll fix those - next week. Let's first get our site translated as it is!

### CODE REVIEW

**Comment by jospoortvliet** [comment]
@LukasReschke Have fun, the rest is yours. I'll not be available tomorrow or this weekend, at least not much - traveling. Ping me once ppl can get nuts on Transifex so I can let our translator know.

File: page-changelog.php [hunk 1]
```diff
@@ -135,109 +201,170 @@
...
 
-<h4>External User Backend</h4>
+<h4><?php
+ echo $l->t('External User Backend');?></h4>
 <h5>Bug fixes</h5>
 <ul>
-	<li>Detect anonymous auth on the server and ignore it (<a href="https://github.com/nextcloud/apps/pull/10">apps/10</a>)</li>
+	<li><?php
+ echo $l->t('Detect anonymous auth on the server and ignore it (<a href="https://github.com/nextcloud/apps/pull/10">apps/10</a>)');?></li>
 </ul>
```
Comment (LukasReschke) [inline page-changelog.php]: I'm not sure if it really is useful to have our changelog translated… 🙈

**Comment by LukasReschke** [comment]
Imported to Transifex. I also made the user `jospoortvliet` administrator of the project.

Just add them to the translators team at https://www.transifex.com/nextcloud/nextcloud-website/language/de/ and then they can start translating.

The resources are named like the URL if you remove the `.json`, so that way a translator can check the context of a string easily. There are two exemptions that are used globally:

- footer.json
  - For the footer
- header-navbar.json
  - For the navbar menu in the top

File: page-changelog.php (hunk 1 above)
Comment (jospoortvliet) [inline page-changelog.php]: yeah, that was not a smart idea. I can remove it?!?!?

later ;-)

**Comment by Espina2** [comment]
When you can give me permissions and I start help translate for Portuguese. :)

**Comment by jospoortvliet** [comment]
for the website we'll let the company take care of that, for both legal and practical reasons, if you don't mind ;-)

**Comment by Espina2** [comment]
I don't mind at all. :) if you need help in Portuguese just let me know. :)

**Pull request merged**